test/localenv/symphony-ghcr-values.yaml

```diff
@@ -9,4 +9,7 @@ symphonyImage:
 paiImage:
   pullPolicy: Never
   repository: ghcr.io/eclipse-symphony/symphony-api
-installServiceExt: false
\ No newline at end of file
+installServiceExt: false
+api:
```

Comment:
We need to test both materialize with label and materialize without label in OSS.

Reply:
Added no label test, too